Fix:centering search

mujohn26 opened the pull request:
#### What does this PR do?
- Applying feedback given about navbar

#### Description of Task to be completed?
- Centered search bar 
-  Put icons in the navbar to the rightmost side of navbar
- Updated README file
#### How should this be manually tested?
- clone the repository
- open index.html in your browser

#### Any background context you want to provide?
- N/A

#### Screenshots (if appropriate)
<img width="1413" alt="Screen Shot 2020-05-04 at 11 45 53" src="https://user-images.githubusercontent.com/52497006/80954192-31cd4580-8dfd-11ea-93ff-6edc11628f86.png">
<img width="1437" alt="Screen Shot 2020-05-04 at 11 46 02" src="https://user-images.githubusercontent.com/52497006/80954213-3b56ad80-8dfd-11ea-9bb4-bc2cd689f13f.png">

#### Questions:
- N/A

mujohn26 committed:
FIXING NAVBAR

Dandush03 commented:
# :red_circle: Status Require: changes :muscle: 
### Hey :wave: 
## Good attempts on this one :muscle: :+1: 

## Here are the issues to address to get the project meet the requirement:wink:

---

##### Header

- [ ] Left menu is missing
- [ ] YouTube Logo isn't the same as the project
- [ ] The right icons are not well aligned, They should have the same size.
- [ ] Add some padding to the right icons

##### Required:
![Screenshot from 2020-04-30 18-04-52](https://user-images.githubusercontent.com/51087936/80763810-57acce80-8b0d-11ea-8dd4-1ba5807cf1f6.png)

##### Yours:
![Screenshot from 2020-04-30 18-04-35](https://user-images.githubusercontent.com/51087936/80763805-554a7480-8b0d-11ea-878c-507eccea2309.png)

---

##### recommended videos

- [ ] Title should be aligned in the center of this section
- [ ] Missing "Up next" and "AUTOPLAY" text
- [ ] Videos have wrong width/height ratio

##### Required:
![Screenshot from 2020-04-30 18-03-53](https://user-images.githubusercontent.com/51087936/80763829-62fffa00-8b0d-11ea-9e8c-6240c493ed37.png)

##### Yours:
![Screenshot from 2020-04-30 18-04-09](https://user-images.githubusercontent.com/51087936/80763832-65faea80-8b0d-11ea-995e-0794ea899e91.png)

---

##### Main Video Title

- [ ] Please make sure to display it the same way, it should have the number of likes and dislike, all element should be aligned
- [ ] Missing Hashtags info
- [ ] Wrong text sizes

##### Required:
![Screenshot from 2020-04-30 18-04-52](https://user-images.githubusercontent.com/51087936/80763851-6f845280-8b0d-11ea-9986-2d6f90a23fe5.png)

##### Yours:
![Screenshot from 2020-04-30 18-04-35](https://user-images.githubusercontent.com/51087936/80763855-714e1600-8b0d-11ea-9ec4-93682f424384.png)

---

##### Comment Section

- [ ] Comment section makes sure you fix it before submitting a new review

##### Required:
![Screenshot from 2020-04-30 18-05-06](https://user-images.githubusercontent.com/51087936/80763865-77dc8d80-8b0d-11ea-8d64-d605fa7cd5ec.png)

##### Yours:
![Screenshot from 2020-04-30 18-05-22](https://user-images.githubusercontent.com/51087936/80763871-79a65100-8b0d-11ea-9d41-5f6a7938816b.png)

---

## Kindly address all these changes and submit a new code review

# __[IMPORTANT]__

## Please do not close this PR and keep working in this same branch so my colleges could track better your improvements 

### Happy coding:computer: 

#### Daniel Laloush

rammazzoti2000 reviewed (changes requested):
### Hello @mujohn26 and @Guy-Gustave, I'm Alex, your reviewer for this PR :man_astronaut: 

First of all, congrats in working on your first Microverse project :star2: 

I see you made some progress from the past reviews but there are still some issues to fix. Rest assured that when you get used to the general requirements for this module the next projects code reviews will be way easier :muscle: 
######
# Required Changes :building_construction: :pray: 
- [ ] Html code must pass all validations [Screenshot from Pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/validation_requirement.png) by using this [validator](https://validator.w3.org/#validate_by_upload)
- [ ]  Pay attention to the vertical size of the search input and button, they should be aligned like the original.
### Clone:![image](https://user-images.githubusercontent.com/3244654/80856371-c98a3280-8c49-11ea-83f5-3886b5436cc7.png)
### Correct navbar:
![image](https://user-images.githubusercontent.com/3244654/80856393-fcccc180-8c49-11ea-9eec-910d08ab71b2.png)
- [ ] The video section should show the avatar of the user who uploaded the video 
([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/video_avatar_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/48945837/))
![image](https://user-images.githubusercontent.com/3244654/80856762-fc81f580-8c4c-11ea-8b1e-5a913137f6b7.png)
- [ ] The comment section is still broken, please fix it before submitting a new review
### Clone:
![image](https://user-images.githubusercontent.com/3244654/80856857-d6108a00-8c4d-11ea-8356-91017d3711d2.png)
### Original comment section: 
![image](https://user-images.githubusercontent.com/3244654/80856696-53d39600-8c4c-11ea-8f7f-9a7799d31e30.png)
- [ ] Recommended videos should be aligned to the right side of the page. Also remove extra space between recommended videos and their description:
![image](https://user-images.githubusercontent.com/3244654/80856940-83839d80-8c4e-11ea-983c-de149f92af93.png)
#
- [ ] [OPTIONAL] Instead of getting images with a background color or taking screenshots of icons use actual icons(). <em>You can find them in sites like [fontaweome](https://github.com/rida-elbahtouri/rida-aye-youtube/pull/fontawesome.com/), they are easier to style, you style icons like text, and they have a transparent background.
In case you don't want to use icons you can also search for png images with transparent background on google images or svg images.</em>

[Read this article if you are not sure which type of image you should use for each case in web development.](https://www.pagecloud.com/blog/web-images-png-vs-jpg-vs-gif-vs-svg)

[Read this article if you want to know more about SVG images, they are great for logos!](https://css-tricks.com/using-svg/)
#
### <em>Kindly fix the REQUIRED issues and send another code review request.

Cheers and Happy coding! :clap: 

Code review by Alexandru Bangau

(Feel free to contact me on slack by the handle of alexandru if you have any additional question)</em>

rammazzoti2000 reviewed (changes requested):
#### Status: Changes Required

mujohn26 commented:
I resolved all the required changes as you see in the screenshot above on comment section it's not breaking on my PC it is running perfectly.

Guy-Gustave commented:
When I will go back home I will check. I am in way

Le sam. 2 mai 2020 à 14:54, MUGIRANEZA JOHN <notifications@github.com> a
écrit :

> I resolved all the required changes as you see in the screenshot above on
> comment section it's not breaking on my PC it is running perfectly.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/mujohn26/video-and-image-viewer/pull/9#issuecomment-622949303>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANQ75DLKK36NCSANUXXNLPDRPQJX3ANCNFSM4MVUTJEA>
> .
>

rammazzoti2000 commented:
> I resolved all the required changes as you see in the screenshot above on comment section it's not breaking on my PC it is running perfectly.

This is the :point_right:  [link](https://rawcdn.githack.com/mujohn26/video-and-image-viewer/feature/feedback-navbar/index.html) to the webpage of this branch (feature/feedback-navbar) that you made the pull request for
You must still fix the recommended videos and the gap alignment between thumbnails and description.
As for your comment section is missing the avatar for each user's comment; I can see the extra space where the broken images were before, but I don't see any image.

mujohn26 commented:
Alex, you can check well on that link you sent to me the avatar on user
comment are there and also the size between thumbnails and descriptions I
have reduced it.


On Sat, May 2, 2020 at 3:21 PM Alex <notifications@github.com> wrote:

> I resolved all the required changes as you see in the screenshot above on
> comment section it's not breaking on my PC it is running perfectly.
>
> This is the 👉 link
> <https://rawcdn.githack.com/mujohn26/video-and-image-viewer/feature/feedback-navbar/index.html>
> to the webpage of this branch (feature/feedback-navbar) that you made the
> pull request for
> You must still fix the recommended videos and the gap alignment between
> thumbnails and description.
> As for your comment section is missing the avatar for each user's comment;
> I can see the extra space where the broken images were before, but I don't
> see any image.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/mujohn26/video-and-image-viewer/pull/9#issuecomment-622952807>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AMQQU3QRSR3WNXK25G2CMTLRPQM4PANCNFSM4MVUTJEA>
> .
>

mujohn26 commented:
The reason why I'm telling that it is because if you check that screenshot in PR descriptions that is how it looks like in my browser.

rammazzoti2000 commented:
> Alex, you can check well on that link you sent to me the avatar on user comment are there and also the size between thumbnails and descriptions I have reduced it.
> […](#)
> On Sat, May 2, 2020 at 3:21 PM Alex ***@***.***> wrote: I resolved all the required changes as you see in the screenshot above on comment section it's not breaking on my PC it is running perfectly. This is the  link <https://rawcdn.githack.com/mujohn26/video-and-image-viewer/feature/feedback-navbar/index.html> to the webpage of this branch (feature/feedback-navbar) that you made the pull request for You must still fix the recommended videos and the gap alignment between thumbnails and description. As for your comment section is missing the avatar for each user's comment; I can see the extra space where the broken images were before, but I don't see any image. — You are receiving this because you were mentioned. Reply to this email directly, view it on GitHub <[#9 (comment)](https://github.com/mujohn26/video-and-image-viewer/pull/9#issuecomment-622952807)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AMQQU3QRSR3WNXK25G2CMTLRPQM4PANCNFSM4MVUTJEA> .

I was using the link for production in rawhack website. Apparently using it for development it shows all the pictures. I apologize for that.
Anyway, you guys did a great job as a first project and you should be proud.
The only thing now left is to wait for the next reviewer assigned for this project to give you feedback.
:raised_hands: I apologize once again, wish you good luck with your next projects

Guy-Gustave commented:
Thanx bro, see today you tried your best, I say thank you! I saw all the
mails. See you tomorrow.

Le sam. 2 mai 2020 à 22:32, MUGIRANEZA JOHN <notifications@github.com> a
écrit :

> Alex, you can check well on that link you sent to me the avatar on user
> comment are there and also the size between thumbnails and descriptions I
> have reduced it.
>
>
> On Sat, May 2, 2020 at 3:21 PM Alex <notifications@github.com> wrote:
>
> > I resolved all the required changes as you see in the screenshot above on
> > comment section it's not breaking on my PC it is running perfectly.
> >
> > This is the 👉 link
> > <
> https://rawcdn.githack.com/mujohn26/video-and-image-viewer/feature/feedback-navbar/index.html
> >
> > to the webpage of this branch (feature/feedback-navbar) that you made the
> > pull request for
> > You must still fix the recommended videos and the gap alignment between
> > thumbnails and description.
> > As for your comment section is missing the avatar for each user's
> comment;
> > I can see the extra space where the broken images were before, but I
> don't
> > see any image.
> >
> > —
> > You are receiving this because you were mentioned.
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/mujohn26/video-and-image-viewer/pull/9#issuecomment-622952807
> >,
> > or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/AMQQU3QRSR3WNXK25G2CMTLRPQM4PANCNFSM4MVUTJEA
> >
> > .
> >
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/mujohn26/video-and-image-viewer/pull/9#issuecomment-623009558>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANQ75DLA7Z6UXGYOWX7NVWLRPR7NRANCNFSM4MVUTJEA>
> .
>

trekab commented:
## Changes Required!
Hello @Guy-Gustave @mujohn26, well done with the requested changes! :clap: You are almost there :wink:  You have a few changes to make:
- [ ] You still have commented code in your CSS file.[Read more](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)
- [ ] [Your clone](https://rawcdn.githack.com/mujohn26/video-and-image-viewer/feature/feedback-navbar/index.html)'s navigation has big top margin. Please reduce it to appear like the [original site](https://www.youtube.com/watch?v=YfOohn8_l90)
- [ ] There is unnecessary big white space between the main video section and the side bar recommended videos.

__Kindly work on the above changes and submit another code review request__
_Feel free to contact me on slack for any inquiries about this review_
__Treasure Kabareebe__

mujohn26 commented:
I fixed it 
-  I reduced white space at the top navbar
-  I reduced space between the main videos and recommended videos
- I deleted commented lines in CSS file

santiago-dev96 commented:
# Pull Request under review by @santiago-rodrig (TSE)

santiago-dev96 commented on index.html:
```diff
@@ -12,18 +12,40 @@
+
+                <img class="menu-icon" src="./images/menu-icon.png" height="20" alt=""/>
+            </div>
+            &nbsp;&nbsp;
```
Completely unnecessary, please use margins with CSS.

santiago-dev96 commented on index.html:
```diff
@@ -12,18 +12,40 @@
 <body>
     <nav class="navbar">
```
This navigation bar could use some padding.

santiago-dev96 commented on index.html:
```diff
@@ -12,18 +12,40 @@
+        </div>
+        <div class="search-container">
+
+            <button type="button" name="button" class="search-button"><img src="./images/search2.png" alt=""
+                class="search-icon" height="17" /></button>
```
Why the `<img/>` and the `<button></button>` have the same class?

santiago-dev96 commented on index.html:
```diff
@@ -12,18 +12,40 @@
+
+            <button type="button" name="button" class="search-button"><img src="./images/search2.png" alt=""
+                class="search-icon" height="17" /></button>
+            </div>
```
Where is the matching `<div>`?

santiago-dev96 commented on index.html:
```diff
@@ -12,18 +12,40 @@
+
+                <img src="https://img.icons8.com/metro/26/000000/appointment-reminders.png" alt="" />
+            </div>
+            &nbsp;&nbsp;&nbsp;
```
Don't use this to put space between elements, it is a bad practice.

santiago-dev96 commented on index.html:
```diff
@@ -12,18 +12,40 @@
+
+                <img src="https://img.icons8.com/material/24/000000/circled-menu.png" alt="" />
+            </div>
+            &nbsp;&nbsp;&nbsp;
```
Don't.

santiago-dev96 commented on index.html:
```diff
@@ -33,20 +55,22 @@
             <iframe class="iframe-main-tag" src="https://www.youtube.com/embed/V74l_zS1x8E"></iframe>
             <br><br>
```
Don't. Use margins with CSS instead.

santiago-dev96 commented on index.html:
```diff
@@ -33,20 +55,22 @@
             <div class="video-title">
-                Learn HTML
+                <a class="hashtag" href="#">#developer #html</a>
+                <br>
```
Same as above.

santiago-dev96 commented on index.html:
```diff
@@ -33,20 +55,22 @@
+                            <img src="./images/dislike.png" alt="" /> 1k
                         </a>
                     </div>
                     &nbsp;&nbsp;&nbsp;
```
Don't.

santiago-dev96 commented on index.html:
```diff
@@ -63,7 +87,7 @@
-                        <img src="./images//user.png" alt="" />
+                        <img src="./images/avatar-icon.png" alt="" />
                     </div>
                     &nbsp;&nbsp;
```
Don't.

santiago-dev96 commented on index.html:
```diff
@@ -143,7 +167,15 @@
+                    Autoplay
+                </div>
+            </div>
+            <br>
```
Stop using this.

santiago-dev96 reviewed (approved):
# Approved! But...
Your code looks messy, try to improve the way your codebase looks for future projects. The sidebar should not have `iframe` elements for the video previews, with regular images will suffice (take a look at the [project requirements for The Odin Project](https://www.theodinproject.com/courses/html5-and-css3/lessons/embedding-images-and-video)). Still! You meet with the requirements for approval. Good luck and have a nice day :wave:.
